src/components/organisms/NavBarSchedule/NavBarSchedule.tsx

```diff
@@ -84,26 +96,63 @@ export const NavBarSchedule: React.FC<NavBarScheduleProps> = ({
   } = useVenueEvents({
     venueIds: relatedVenueIds,
   });
-
   const isLoadingSchedule = isLoading || isEventsLoading;
 
   const [selectedDayIndex, setSelectedDayIndex] = useState(0);
 
+  const minDate = useMemo(() => {
+    if (relatedVenueEvents.length) {
+      return Math.min(
+        ...relatedVenueEvents.map((event) =>
+          secondsToMilliseconds(
+            event.start_utc_seconds + minutesToSeconds(event.duration_minutes)
+          ) > startOfToday().getTime()
+            ? event.start_utc_seconds
+            : Number.MAX_SAFE_INTEGER
```

Comment:
and also, maybe instead of this check we should better create a const `liveAndFutureEvents` (instead of `relatedVenueEvents`) - and iterate through them everywhere? as we won't need the events that have already ended.

Reply:
moved it into a separate var 365daad